operator/controllers/config/constants.go

```diff
@@ -151,6 +151,7 @@ const (
 	EnvNodePublishMethodKey           = "NODEPUBLISH_METHOD"
 	EnvVolumeStatsCapabilityKey       = "VOLUME_STATS_CAPABILITY"
 	EnvDiscoverCGFilesetKey           = "DISCOVER_CG_FILESET"
+	NetworkPolicyKey                  = "NETWORK_POLICY"
```

Comment:
I think the env var name should be something like `HOST_NETWORK` instead of `NETWORK_POLICY`. @deeghuge what do you think?

Reply:
true @amdabhad  , as value set to NETWORK_POLICY variable and assigned to host network will be opposite

Reply:
Here we need to know which feature is enabled with env and not what the functionality we does inside. It's like we check for e.g. we check whether it's openshift cluster or not but what we do inside logic based on that is different. On seeing logs or config map one should know network policy feature is enabled or not and not about the host network that is internals what we do based on this env.

Reply:
updated to new env. @saurabhwani5 Please test with new env HOST_NETWORK with same values ENABLED/DISABLED.
Latest image: quay.io/hemalatha_gajendran/tunable_host_network:latest

Reply:
@saurabhwani5, please hold the testing till further review.. as we need some more changes, e.g. @hemalathagajendran we need default value of HOST_NETWORK as ENABLED now